GLTFSDK/Inc/GLTFSDK/MeshPrimitiveUtils.h

```diff
@@ -57,6 +57,18 @@ namespace Microsoft
 
             std::vector<uint32_t> GetJointWeights32(const Document& doc, const GLTFResourceReader& reader, const Accessor& accessor);
             std::vector<uint32_t> GetJointWeights32_0(const Document& doc, const GLTFResourceReader& reader, const MeshPrimitive& meshPrimitive);
+
+            std::string SerializeTriangulatedIndices16(const uint16_t* indices, size_t indexCount, MeshMode mode, BufferBuilder& bufferBuilder);
```

Comment:
These utils shouldn't use BufferBuilder at all. Return a vector of indices and let the user decide what to do with it. Currently it's making too many assumptions about how the user wants to serialize their data, i.e. in it's own BufferView, not interleaved, etc....

Reply:
Sure.